css/header.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans&display=swap");
+
+* {
+    box-sizing: border-box;
+    font-family: "PT Sans", sans-serif;
+}
+
+.header {
+    padding: 1em 0;
+    padding-bottom: 0.7em;
```

Comment:
you can use shorthand
padding: 1em 0 0.7em 0;

Reply:
Ack, thanks for the recommendation